doc/whats_new.rst

```diff
@@ -1,6 +1,112 @@
 0.5.0
 =====
 
+    **Released November 2018**
+
+NEW
+---
+
+  :ref:`interactive plotting functions <interactive-plotting>`,
+  eg for use in a notebook.
+
+- New functions :func:`nilearn.plotting.view_surf` and
+  :func:`nilearn.plotting.view_surf` and
+  :func:`nilearn.plotting.view_img_on_surf` for interactive visualization of
+  maps on the cortical surface in a web browser.
+
+- New functions :func:`nilearn.plotting.view_connectome` and
+  :func:`nilearn.plotting.view_markers` to visualize connectomes and
+  seed locations in 3D
+
+- New function :func:`nilearn.plotting.view_img` for interactive
+  visualization of volumes with 3 orthogonal cuts.
+
+:Note: :func:`nilearn.plotting.view_img` was `nilearn.plotting.view_stat_map` in preview builds.
+
+- :func:`nilearn.plotting.find_parcellation_cut_coords` for
+  extraction of coordinates on brain parcellations denoted as labels.
+
+- Added :func:`nilearn.plotting.find_probabilistic_atlas_cut_coords` for
+  extraction of coordinates on brain probabilistic maps.
+
+
+**Minimum supported versions of packages have been bumped up.**
+  - scikit-learn -- v0.18
+  - scipy -- v0.17
+  - pandas -- v0.18
+  - numpy -- v1.11
+  - matplotlib -- v1.5.1
+
+**Nilearn Python2 support is being removed in the near future.**
+  Users with a Python2 environment will be warned
+  at their first Nilearn import.
+
+**Additional dataset downloaders for examples and tutorials.**
+
+- :func:`nilearn.datasets.fetch_surf_fsaverage`
+- :func:`nilearn.datasets.fetch_atlas_pauli_2017`
+- :func:`nilearn.datasets.fetch_neurovault_auditory_computation_task`
+- :func:`nilearn.datasets.fetch_neurovault_motor_task`
+
+
+ENHANCEMENTS
+------------
+
+ :func:`nilearn.image.clean_img` now accepts a mask to restrict
+ the cleaning of the image, reducing memory load and computation time.
+
+ NiftiMaskers now have a `dtype` parameter, by default keeping the same data type as the input data.
+
+ Displays by plotting functions can now add a scale bar (see :ref:`plotting`)
+
+
+IMPROVEMENTS
+------------
+
+ - Lots of other fixes in documentation and examples.
+ - A cleaner layout and improved navigation for the website, with a better introduction.
+ - Dataset fetchers are now  more reliable, less verbose.
+ - Searchlight().fit() now accepts 4D niimgs.
+ - Anaconda link in the installation documentation updated.
+ - Scipy is listed as a dependency for Nilearn installation.
+
+Notable Changes
+---------------
+
+ Default value of `t_r` in :func:`nilearn.signal.clean` and
+ :func:`nilearn.image.clean_img` is now 2.5
+ and cannot be None if `low_pass` or `high_pass` is specified.
+
+**LOTS of changes and improvements. Detailed change list follows.**
+
+0.5.0 rc
```

Comment:
Do we need this ?

Reply:
The enthusiasts who like to install alpha and beta builds would want to know what changed from beta to stable, which is covered under `rc`. Early testers often hunger for extra details in the change log. Doing these things helps maintain their ongoing engagement, and gives us a steady pool of alpha-beta testers.

The enthusiasts, passionate about Nilearn but not beta testers upgrading from 0.4.2 to 0.5.0 would like to quickly see all the best new things improved and to try out. For that I have an overarching summary, under 0.5.0 . 

For many remaining users, they won't care if we don't include a What's New, so this isn't for them at all.